tools.go

```diff
@@ -1,3 +1,4 @@
+//go:build tools
```

Comment:
Have you upgraded golang to v1.17+ locally?

Reply:
yes